AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1291,6 +1291,8 @@ quotev.com#$#body #res_area ::selection { background-color: #3297FD!important; }
 !
 !--- JS ----
 !
+brunch.co.kr#%#//scriptlet('prevent-addEventListener', 'keydown')
+brunch.co.kr,page.kakao.com,fmkorea.com#%#//scriptlet('prevent-addEventListener', 'contextmenu')
 minepuls.tistory.com#%#//scriptlet('remove-attr', 'onselectstart|ondragstart|oncontextmenu|onpaste|oncopy', 'body')
 page.kakao.com,fmkorea.com#%#//scriptlet('prevent-addEventListener', 'contextmenu')
```

Comment:
`page.kakao.com,fmkorea.com#%#//scriptlet('prevent-addEventListener', 'contextmenu')`
covered by 
`brunch.co.kr,page.kakao.com,fmkorea.com#%#//scriptlet('prevent-addEventListener', 'contextmenu')`

Reply:
Please wait.

Reply:
Is it Okay?
https://github.com/AdguardTeam/AdguardFilters/pull/87656/commits/bf92f501442c35ee0e3d29f6a265c9fb56d54222